src/less/components/payex/button.less

```diff
@@ -33,7 +33,7 @@
 		}
 
 		&:active {
-			border: 1px solid @brand-secondary-info-1;
+			border: 1px solid @brand-primary;
```

Comment:
remember to try to use CSS custom properties when possible and when it is not requiring too much work 😉